FIRO_TSEnsembles/src/main/java/hec/stats/CumulativeFlow.java

```diff
@@ -0,0 +1,44 @@
+package hec.stats;
+
+public  class CumulativeFlow implements Computable, Configurable {
```

Comment:
The issue seems to be a missing interface method definition for units on the Configurable interface. See issue #78